README.md

```diff
@@ -27,18 +27,20 @@ Follow up instructions located [here](https://github.com/MoveOnOrg/Spoke/blob/ma
 3. `npm install`
 4. `npm install -g foreman`
 5. `cp .env.example .env`
-6. Run `npm run dev` to create and populate the tables.
-7. Create an [Auth0](https://auth0.com) account. In your Auth0 account, go to Settings -> Clients -> and then grab your Client ID, Client Secret, and your Auth0 domain (should look like xxx.auth0.com). Add those inside your `.env` file (AUTH0_CLIENT_ID, AUTH0_CLIENT_SECRET, AUTH0_DOMAIN respectively).
+6. Create an [Auth0](https://auth0.com) account. In your Auth0 account, go to Settings -> Clients -> and then grab your Client ID, Client Secret, and your Auth0 domain (should look like xxx.auth0.com). Add those inside your `.env` file (AUTH0_CLIENT_ID, AUTH0_CLIENT_SECRET, AUTH0_DOMAIN respectively).
```

Comment:
Can we use markdown-style numbering and number every bullet point as `1.`? The items will render the same way, but we won't have to spend time renumbering if we insert or delete an item early in the list.

Reply:
I personally like the numbering in the file, because then it functions as just a text file as well (I generally read the readme file from the checkout, not the github page).

The renumbering here is because we're flipping the order, not messing with ordering accidents.